conformance/results/pyre/aliases_explicit.toml

```diff
@@ -9,26 +9,6 @@ Incorrectly rejects import alias of `TypeAlias` when used to define type alias.
 Does not report invalid specialization of already-specialized generic type alias.
 """
 output = """
-aliases_explicit.py:23:0 Incompatible variable type [9]: GoodTypeAlias9 is declared to have type `TA` but is used as type `Type[typing.Callable[..., Variable[$synthetic_attribute_resolution_variable]]]`.
```

Comment:
Hmm, it looks like pyre isn't running properly on your system. It should produce errors in these cases. You'll need to determine why it doesn't work on your system.